rosdep/base.yaml

```
@@ -5647,6 +5647,9 @@ tmux:
   fedora: [tmux]
   gentoo: [app-misc/tmux]
   ubuntu: [tmux]
+trang:
+  debian: [trang]
+  ubuntu: [trang]
```

This is packaged for Fedora: https://apps.fedoraproject.org/packages/trang
```suggestion
  fedora: [trang]
  ubuntu: [trang]
```